iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/processor/downsampling/sdt/SwingingDoorTrendingSamplingProcessor.java

```diff
@@ -49,21 +51,7 @@ public class SwingingDoorTrendingSamplingProcessor extends DownSamplingProcessor
    */
   private double compressionDeviation;
 
-  /**
-   * The minimum time distance between two stored data points if current point time to the last
-   * stored point time distance <= compressionMinTimeInterval, current point will NOT be stored
-   * regardless of compression deviation
-   */
-  private long compressionMinTimeInterval;
-
-  /**
-   * The maximum time distance between two stored data points if current point time to the last
-   * stored point time distance >= compressionMaxTimeInterval, current point will be stored
-   * regardless of compression deviation
-   */
-  private long compressionMaxTimeInterval;
-
-  private PartialPathLastObjectCache<SwingingDoorTrendingFilter<?>> pathLastObjectCache;
+  private PartialPathLastObjectCache<SwingingDoorTrendingFilter> pathLastObjectCache;
```

Comment:
extends DownSamplingProcessor\<SwingingDoorTrendingFilter\>? Then the cache can be put above(

Reply:
This conflicts with TumblingTimeSamplingProcessor